src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -320,25 +328,28 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                                     <progress
                                         max=""100""
                                         value=""{System.Math.Pow(amplitude.Magnitude, 2.0) * 100}""
-                                        style=""width: 70%;""
+                                        style=""width: 100%;""
                                     > 
                                     <td>
                                     <p id=""round${count}""> 
                                     <script>
                                     var num = {System.Math.Pow(amplitude.Magnitude, 2.0) * 100};
-                                    num = num.toFixed(2);
-                                     document.getElementById(""round${count}"").innerHTML = num;
+                                    num = num.toFixed({measurementPrecision});
+                                    var num_string = num + ""%"";
+                                     document.getElementById(""round${count}"").innerHTML = num_string;
                                     </script> </p>
                                     </td>
                                 </td>
                             "), // not sure what style width does
+                            //TODO: fix viewing % (might have accomplished this)
                             MeasurementDisplayStyle.NumberOnly => FormattableString.Invariant($@"
                                 <td> 
-                                    <p id=""round${count}""> 
+                                    <p id=""round${count}""; style=""text-align: right""> 
```

Comment:
Small suggestion, since semicolons aren't needed between attributes:
```suggestion
                                    <p id=""round${count}"" style=""text-align: right""> 
```